src/transaction/mvcc_table.cpp

```diff
@@ -707,17 +707,35 @@ mvcctable::get_global_oldest_visible () const
 }
 
 MVCCID
-mvcctable::update_global_oldest_visible ()
+mvcctable::update_global_oldest_visible (const MVCCID pts_oldest_visible /* = MVCCID_LAST */)
 {
-  if (m_ov_lock_count == 0)
+  if (pts_oldest_visible != MVCCID_ALL_VISIBLE)
```

Comment:
```suggestion
  if (pts_oldest_visible != MVCCID_LAST)
```

I think `pts_oldest_visible` can be MVCCID_ALL_VISIBLE,
if PTS is just connected to PS and PTS did not send any mvccid to PS yet.

Reply:
So, I think if pts_oldest_visible is `MVCCID_ALL_VISIBLE`, then it needs to be considered.

Reply:
It's intended. If it is MVCCID_ALL_VISIBLE, it doesn't update the oldest mvccid and, in turn, the vacuum doesn't go forward and will retry in the next execution. Plus, `MVCCID_LAST` is also intended, which means no PTS exists.

Reply:
Oh, OK. Thanks for explain.